가운데-글자-가져오기.js

```diff
@@ -0,0 +1,17 @@
+function solution(s) {
+    //문자열 길이 length
+    //짝수면 가운데 글짜 두개 , 홀수면 가운대 글자 1개
+    
```

Comment:
함수 내에 answer라는 변수명이 있는데 answer변수 선언은 따로 해주지 않았네요?
변수선언 해주시고 commit하실 때 빠트린 것 없는지 항상 확인하고 commit하셔야 합니다
```suggestion
    let answer = '';
```